### ISSUE TITLE
Illegible text

### ISSUE THREAD

**Issue by marcgoblot** [arXiv/html_feedback#3516]
### Description

In dark mode, the blue text becomes inaccessible and needs to change to maintain contrast ration.

### (Required) What is this issue most closely related to? Select one.

Choose One

### Internal issue ID

d58ea4fb-684f-4532-a477-28747a3a0ba0

### Paper URL

https://arxiv.org/html/2408.13296v1

### Browser

Firefox/136.0

### Device Type

MacBook

**Comment by dginev** [comment]
Thank you for your report - the specific issue is now solved.

We have regenerated the article and you should be able to see an improvement.
